gnomad_constraint/resources/resource_utils.py

```diff
@@ -92,6 +103,46 @@ def get_sites_resource(
         return public_release_grch38(data_type).versions[version]
 
 
+def get_coverage_ht(
+    data_type: str,
+    version: str = CURRENT_VERSION,
+) -> TableResource:
+    """
+    Return TableResource of coverage Table.
+
+    :param data_type: One of "exomes", "genomes" or "context.
+    :param version: One of the release versions (`VERSIONS`). Default is
+        `CURRENT_VERSION`.
+    :return: TableResource of coverage Table.
+    """
+    check_param_scope(version, data_type)
+    # TODO: decide path to coverage Table
+    return TableResource("")
+
+
+def get_annotated_context_ht(
+    version: str = CURRENT_VERSION,
+    use_old_data: bool = False,
+) -> TableResource:
+    """
+    Return TableResource of annotated context Table.
+
+    :param version: One of the release versions (`VERSIONS`). Default is
+        `CURRENT_VERSION`.
+    :param use_old_data: Whether to use old annotated context Table. Default is False.
+    :return: TableResource of annotated context Table.
+    """
+    if use_old_data:
+        return TableResource(
+            "gs://gcp-public-data--gnomad/papers/2019-flagship-lof/v1.0/context/Homo_sapiens_assembly19.fasta.snps_only.vep_20181129.ht"
+        )
+
+    check_param_scope(version)
+    return TableResource(
+        f"{constraint_tmp_prefix}/{version}/preprocessed_data/annotated_context.ht"
```

Comment:
The previous dataset was based on the genome build and the vep version, is that something we want to do too?